src/components/InputField.tsx

```diff
@@ -58,6 +60,14 @@ const InputField = ({
           autoCapitalize="none"
           onFocus={() => setIsFocused(true)}
           onBlur={() => setIsFocused(false)}
+          onLayout={() => {
```

Comment:
Blijkbaar (maar kan er geen reports van vinden) kan autoFocus worden uitgevoerd als de component is gemount, maar nog niet attached. In Fabric is de timing van deze events anders. De fix hier is om een expliciete focus te doen op een moment dat we weten dat het ook werkt. Gebeurt eenmalig na mount.

Dit fixt het gedrag van Observation op React Native 0.83.

Reply:
> Ah `ref={inputRef}` mist nog, dat verklaart mijn bug ook wel een beetje.

Beetje suf, toegevoegd.